### ISSUE TITLE
Maven repo credentials are no injected in gradle.build when using "--idea" flag

### ISSUE THREAD

**Issue by jiraguha** [holgerbrandl/kscript#261]
Hi, Thanks for this awesome tool :-). I have a little bug:
When in *.kts you have
```
@file:MavenRepository("id-repo","url-repo", password = "pass", user="user")
```
And use the flag `--idea`, the generated gradle.build.kts is:

```
repositories {
    ...
    maven {
        url = uri("url-repo")
    }
}
```
But it should be

```
repositories {
    ...
    maven {
        url = uri("url-repo")
        credentials {
            username = "user"
            password = "pass"
        }
    }
}
```
Kr,
JP

**Comment by holgerbrandl** [comment]
Indeed a bug. Thanks for spotting it.

**Comment by holgerbrandl** [comment]
Fixed with #262
